Dalamud/Plugin/Internal/PluginManager.cs

```diff
@@ -786,11 +786,7 @@ public void ScanDevPlugins()
             
             Log.Verbose("Scanning dev plugins at {Path}", setting.Path);
 
-            if (Directory.Exists(setting.Path))
-            {
-                devDllFiles.AddRange(new DirectoryInfo(setting.Path).GetFiles("*.dll", SearchOption.AllDirectories));
-            }
-            else if (File.Exists(setting.Path))
```

Comment:
For now, let's keep the recursive loading mode but just block setting folders in UI.

Removing this behavior should be an api11 todo so we have time to announce it.